src/commands/init.ts

```diff
@@ -48,7 +48,7 @@ let choices: {
   },
   {
     value: "apphosting",
-    name: "App Hosting: Configure an apphosting.yaml file for App Hosting",
+    name: "App Hosting: Enable web app deployments with App Hosting",
```

Comment:
This seems wrong to me? are we enabling deployments, or making a backend?  I think the flow as-written doesn't link to github?

Reply:
cc @jpreid7396 my understanding is that the description is meant to be generic enough to cover set up for deploy from source, but also any other. I'll defer to Julia here about the language specifics